httpsign.go

```diff
@@ -59,6 +60,14 @@ func (hs *HttpSign) SignToProxy(h http.Handler, v GetValue) http.Handler {
 func (hs *HttpSign) Verify(h http.Handler, v GetValue) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		header := r.Header.Get(hs.HeaderName)
+		if hs.DisableVerify {
+			if header == "" {
+				hs.writeInvalid(w)
```

Comment:
I think if DisableVerify is true then it should always be a pass through and not even check for the header